index.js

```diff
@@ -112,32 +112,42 @@ function getServices(offset) {
   });
 }
 
+/**
+ * Create TTE
+ * @param {Array} data - service data
+ * @param {Array} pdData - list of pagerduty data
+ * @returns {Object} - A tte
+ */
+function createTTE(data, pdData) {
```

Comment:
if we are using data just with data.service.summary, is there any other reason for passing the whole data here?